app/pandas.ipynb

```diff
@@ -271,24 +696,98 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
     "# Data\n",
     "data1 = {'Name': ['John', 'Emily', 'Ryan'],\n",
     "         'Age': [25, 28, 22]}\n",
     "data2 = {'Name': ['Emily', 'Ryan', 'Mike'],\n",
     "         'City': ['Los Angeles', 'Chicago', 'Houston']}\n",
     "\n",
+    "df1 = pd.DataFrame(data1)\n",
+    "df2 = pd.DataFrame(data2)\n",
+    "df_concat = pd.concat([df1, df2])\n",
+    "df_merged = pd.merge(df1, df2, on=\"Name\", how=\"outer\")\n",
```

Comment:
You have implemented the concatenation and merging of DataFrames correctly. However, consider displaying `df_concat` to compare it with `df_merged` and understand the differences between concatenation and merging.